src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  align-content: center;
```

Comment:
The 'align-content' property has no effect on this flex container since it's not applying flex-wrap. If you intended to center the items vertically, 'align-items' is sufficient.